pkg/validate/account_kubernetes.go

```diff
@@ -0,0 +1,63 @@
+package validate
+
+import (
+	"fmt"
+	"github.com/armory/spinnaker-operator/pkg/apis/spinnaker/v1alpha1"
+	"github.com/armory/spinnaker-operator/pkg/halconfig"
+	"github.com/armory/spinnaker-operator/pkg/validate/configfinder"
+)
+
+type kubernetesAccount struct {
```

Comment:
I'd expect `kubernetesAccount` to be more typed than `Config: interface{}`. Also can you export so it can be reused in account CRDs?

Reply:
I was thinking in growing typing organically when we absolutely need a field for something, for the reason that account config fields are already defined in Halyard codebase, and if they need to be changed (like in adding a new field), then we'll end up with two places to remember to update instead of one.

Reply:
This will change with the merge with account crd work